requirements.txt

```diff
@@ -1,2 +1 @@
-Django>=1.8
```

Comment:
True. However, not everyone seems to use setuptools, and `install_requires` only works with setuptools. Not really a problem to remove it I guess since it should be pretty clear that Django 1.8 or higher is required.